fuel-tx/src/transaction/types/mint.rs

```diff
@@ -51,7 +51,6 @@ impl MintMetadata {
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[derive(fuel_types::canonical::Deserialize, fuel_types::canonical::Serialize)]
 #[canonical(prefix = TransactionRepr::Mint)]
-#[cfg_attr(feature = "typescript", wasm_bindgen::prelude::wasm_bindgen)]
```

Comment:
Why is this removed?

Reply:
The `wasm_bindgen` doesn't know how to work with generics, so it is not possible to inherit for `Create` and `Script` transactions. So, I reworked how we are passing any transaction to typescript.